xbmc/pvr/epg/EpgDatabase.cpp

```diff
@@ -330,6 +330,8 @@ void CPVREpgDatabase::UpdateTables(int iVersion)
   {
     m_pDS->exec("ALTER TABLE savedsearches ADD sIconPath varchar(255);");
     m_pDS->exec("UPDATE savedsearches SET sIconPath = ''");
+    m_pDS->exec("ALTER TABLE epgtags ADD sParentalRatingIcon varchar(512);");
+    m_pDS->exec("ALTER TABLE epgtags ADD sParentalRatingSource varchar(128);");
```

Comment:
This is wrong. You need to bump epg database version to 18. The code that inserts the two new columns MUST be executed in a block `if (iVersion < 18)`.

Reply:
Done.